setup.py

```diff
@@ -27,6 +31,17 @@
     exec(f.read(), about)
 
 
+def _get_data_files():
+    pkg_config_command = 'pkg-config --variable=completionsdir bash-completion'
+    bash_completions_dir = subprocess.getoutput(pkg_config_command)
+    if bash_completions_dir:
+        return [(bash_completions_dir, ['build/flapjack.bash-completion'])]
```

Comment:
I'm not sure this does what it appears to do. Googling suggests that it installs the file inside the EGG. In any case when I install with `pip3 install --user .` it doesn't put the completion file in `/usr/share/bash-completion/completions`.

In addition, the other files in my completions directory don't seem to have the `.bash-completion` extension, does that vary from system to system?

Reply:
Did you see this comment from last time?